.github/CODEOWNERS

```diff
@@ -62,6 +62,7 @@
 /pubsub/**/*                           @GoogleCloudPlatform/api-pubsub-and-pubsublite @GoogleCloudPlatform/python-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers
 /pubsublite/**/*                       @GoogleCloudPlatform/api-pubsub-and-pubsublite @GoogleCloudPlatform/python-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers
 /managedkafka/**/*                     @GoogleCloudPlatform/api-pubsub-and-pubsublite @GoogleCloudPlatform/python-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers
+/dataplex/**/*                         @GoogleCloudPlatform/python-samples-reviewers @GoogleCloudPlatform/cloud-samples-reviewers @GoogleCloudPlatform/googleapi-dataplex
```

Comment:
@m-strzelczyk does [python-docs-samples](https://github.com/GoogleCloudPlatform/python-docs-samples) work differently than [java-docs-samples](https://github.com/GoogleCloudPlatform/java-docs-samples)? In java's CODEOWNERS it was correct to add @GoogleCloudPlatform/googleapi-dataplex as an owner (https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/CODEOWNERS#L92)

Reply:
the team has to be added to the repo. I'll sort that out.

Reply:
the remaining error was with another line in the file, checked and the account is no longer Google affiliated so have removed.